src/components/Adyen/Adyen.js

```diff
@@ -288,17 +289,23 @@ const Adyen = ({
     const isBancontactCard =
       selectedPaymentMethodRef?.current?.methodName === 'bancontact_card';
 
-    let checkbox = document.querySelector(`.checkbox-${methodName}`);
+    let checkbox = document.querySelector(`#checkbox-${methodName}`);
+    let checkboxWrapper = document.querySelector(
```

Comment:
I have changed those classnames but still there's id and class because id goes to `input` element and `class` to its parent, I could add something like`inputId` and `labelId` but it requires more changes also in tests and maybe it's better to do it in other task than in P2